_cxx/envoy.mk

```diff
@@ -7,6 +7,8 @@ YES_I_AM_OK_WITH_COMPILING_ENVOY ?=
 ENVOY_TEST_LABEL ?= //test/...
 # Set RSYNC_EXTRAS=Pv or something to increase verbosity.
 RSYNC_EXTRAS ?=
+# Set to non-empty to enable compiling Envoy in FIPS mode.
+FIPS_MODE ?=
```

Comment:
If it affects the Envoy build, then it's gotta be included in the image tag and in the section of variables that `IF YOU MESS WITH ANY OF THESE VALUES, YOU MUST RUN make update-base.`

Reply:
Updated